app/controllers/api/v0/base_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V0::BaseController < ApplicationController
+  include OpenidConnect::ProtectedResourceEndpoint
+
+  def user
+    current_token ? current_token.authorization.user : nil
+  end
```

Comment:
Actually let's rename this to `current_user` to be in line with the non API controllers.